### PR TITLE
Doc: Add python_version value

### PR DESCRIPTION
**Pull request by tapaswenipathak**
### The issue

Fixes https://github.com/pypa/pipenv/issues/2318.

### The fix

Defined value of `python_version`.

### The checklist

* [X] Associated issue
* [X] A news fragment in the `news/` directory to describe this fix with the extension `.bugfix`, `.feature`, `.behavior`, `.doc`. `.vendor`. or `.trivial` (this will appear in the release changelog).

### CODE REVIEW

File: docs/basics.rst [hunk 1]
```diff
@@ -125,7 +125,7 @@ Example Pipfile.lock
 
 - Generally, keep both ``Pipfile`` and ``Pipfile.lock`` in version control.
 - Do not keep ``Pipfile.lock`` in version control if multiple versions of Python are being targeted.
-- Specify your target Python version in your `Pipfile`'s ``[requires]`` section. Ideally, you should only have one target Python version, as this is a deployment tool.
+- Specify your target Python version in your `Pipfile`'s ``[requires]`` section. Ideally, you should only have one target Python version, as this is a deployment tool. ``python_version`` can be ``>`` Python 2.7 and ``<`` Python 3.7.
```
Comment (frostming) [inline docs/basics.rst]: This line is confusing, people may think he should define `python_version = ">2.7"` but we do not support this.

We shouldn't constrain the value of `python_version`, it should be working for any Python version that virtualenv supports, and no upper bounds for future python versions.

Comment (tapaswenipathak) [reply docs/basics.rst]: Can you please write what you think you know as proper statement? Did you read the ticket, ticket clearly says the user didn't know the supported Python version numbers, right?

Comment (frostming) [reply docs/basics.rst]: I think the point is user doesn't know the valid value format of that field.
For `python_version`, `X.Y` is the proper value, without other binary operators, for `python_full_version`, it is `X.Y.Z`

Comment (tapaswenipathak) [reply docs/basics.rst]: Perfect.

File: news/3913.doc.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Define python_version value.
```
Comment (frostming) [inline news/3913.doc.rst]:
```suggestion
Clarify the proper value of `python_version`.
```

Comment (tapaswenipathak) [reply news/3913.doc.rst]: One more variable has been documented, can't accept the edit, ignoring.

**Comment by tapaswenipathak** [comment]
@frostming: anything remaining?

**Comment by tapaswenipathak** [comment]
@frostming: anything remaining?

**Comment by frostming** [comment]
@tapaswenipathak No, thanks for contributing

**Pull request merged**